<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLD_ItemProps</class>
 <widget class="QDockWidget" name="WLD_ItemProps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/toolbar/toolbar.qrc">
    <normaloff>:/toolbar/dock/world16.png</normaloff>:/toolbar/dock/world16.png</iconset>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Item Properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents_2">
   <layout class="QGridLayout" name="WLDItemPropsGrid">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="WLD_LevelGrp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Level</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_16" columnstretch="0,100,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="11" column="1" colspan="2">
        <widget class="QSpinBox" name="WLD_PROPS_EnterTo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="WLD_PROPS_lvlPos">
         <property name="text">
          <string notr="true">Pos: [x. y]</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Go to coordinates:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,100">
          <item row="1" column="0">
           <widget class="QLabel" name="label_54">
            <property name="text">
             <string notr="true">Y:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_53">
            <property name="text">
             <string notr="true">X:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="WLD_PROPS_GetPoint">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="WLD_PROPS_GotoY">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="WLD_PROPS_GotoX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="WLD_PROPS_PathBG">
         <property name="text">
          <string>Path background</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLabel" name="label_52">
         <property name="text">
          <string>Level title</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="WLD_PROPS_BigPathBG">
         <property name="text">
          <string>Big path background</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLineEdit" name="WLD_PROPS_LVLFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QLineEdit" name="WLD_PROPS_LVLTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_56">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Enter to door ID:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="WLD_PROPS_AlwaysVis">
         <property name="text">
          <string>Always visible</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="WLD_PROPS_lvlID">
         <property name="text">
          <string notr="true">LevelID: [z]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="WLD_PROPS_GameStart">
         <property name="text">
          <string>Game start point</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="WLD_PROPS_LVLBrowse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="3">
        <widget class="QGroupBox" name="group_pathOpens">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>145</height>
          </size>
         </property>
         <property name="title">
          <string>Open path by exists:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_17" columnstretch="100,0,0,0,100">
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="icon_exit_directions">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>You can set a condition to open path in specified direction (controls at each side of those arrows), dependent on level exit code.</string>
            </property>
            <property name="pixmap">
             <pixmap resource="../../_resources/arrows/arrows.qrc">:/arrows/box.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="WLD_PROPS_ExitLeft">
            <property name="text">
             <string notr="true">L</string>
            </property>
            <property name="icon">
             <iconset resource="../../_resources/arrows/arrows.qrc">
              <normaloff>:/arrows/red_left.png</normaloff>:/arrows/red_left.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QToolButton" name="WLD_PROPS_ExitRight">
            <property name="text">
             <string notr="true">R</string>
            </property>
            <property name="icon">
             <iconset resource="../../_resources/arrows/arrows.qrc">
              <normaloff>:/arrows/red_right.png</normaloff>:/arrows/red_right.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" rowspan="2" colspan="5">
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="space_top_left">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="WLD_PROPS_ExitTop">
               <property name="text">
                <string notr="true">T</string>
               </property>
               <property name="icon">
                <iconset resource="../../_resources/arrows/arrows.qrc">
                 <normaloff>:/arrows/red_up.png</normaloff>:/arrows/red_up.png</iconset>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="space_top_right">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="5">
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="space_bottom_left">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="WLD_PROPS_ExitBottom">
               <property name="text">
                <string notr="true">B</string>
               </property>
               <property name="icon">
                <iconset resource="../../_resources/arrows/arrows.qrc">
                 <normaloff>:/arrows/red_down.png</normaloff>:/arrows/red_down.png</iconset>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="space_bottom_right">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Level file:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_17">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../_resources/arrows/arrows.qrc"/>
  <include location="../../_resources/toolbar/toolbar.qrc"/>
 </resources>
 <connections/>
</ui>
